### PR TITLE
Large test and fix

### LINKED ISSUE #14: Add integration test for reading and writing from real Spanner instance
Something more polished than the [wip](https://github.com/elefeint/r2dbc-wip/blob/49cd39afccf7f83cb0c1682611cadcef517f884d/src/test/java/io/r2dbc/spanner/SampleSpannerTest.java#L86).

### LINKED ISSUE #104: Bug: Codec binding ARRAYs is not specifying inner element type
Discovered while writing integration test.

Will fix along with test.

### PR DESCRIPTION
**Pull request by ChengyuanZhao**
fixes #104  
fixes #14 

writes multi-MB rows and fixes an issue writing array-columns and tests

### CODE REVIEW

File: src/test/java/com/google/cloud/spanner/r2dbc/statement/StatementBindingsTest.java [hunk 1]
```diff
@@ -32,18 +33,25 @@ public void addBasicBinding() {
...
+          entry("age", Type.newBuilder().setCode(TypeCode.INT64).build())
+      );
 
     assertThat(statementBindings.getBindings())
         .containsExactly(
             Struct.newBuilder()
                 .putFields("name", Value.newBuilder().setStringValue("John").build())
                 .putFields("age", Value.newBuilder().setStringValue("50").build())
+                .putFields("array", Value.newBuilder().setListValue(
+                    ListValue.newBuilder().addValues(Value.newBuilder().setStringValue("a").build())
```
Comment (dzou) [inline src/test/java/com/google/cloud/spanner/r2dbc/statement/StatementBindingsTest.java]: For building List values, i would suggest making a helper method in the test for this.

Comment (ChengyuanZhao) [reply src/test/java/com/google/cloud/spanner/r2dbc/statement/StatementBindingsTest.java]: I totally would if this wasn't just used in one place though

File: .travis.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ install:
 
 script:
   - if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ]; then
-      ./mvnw verify -B -V;
+      ./mvnw verify -B -V -q;
```
[the remark this answers is not in the document]
Comment (elefeint) [reply .travis.yml]: yep, ignore this

**Comment by ChengyuanZhao** [comment]
@dmitry-s  PTAL

**Pull request merged**